muxtools/utils/parsing.py

```diff
@@ -214,7 +214,7 @@ def parse_chapters_bdmv(
                         warn("Couldn't parse fps from playlist! Will take fps from source clip.", parse_chapters_bdmv)
                         fps = clip_fps
 
-                    resolved_ts = resolve_timesource_and_scale(fps, TimeScale.M2TS, allow_warn=False, caller=parse_chapters_bdmv)
+                    resolved_ts = resolve_timesource_and_scale(fps, TimeScale.MKV, allow_warn=False, caller=parse_chapters_bdmv)
```

Comment:
You shouldn't do that.
There should be 2 ABCTimestamps objects.
One with the m2ts timescale and the other one with the mkv timescale.
The m2ts timescale object will be used to get the frame from the time.
The mkv timescale will object will be used to get the time from the frame.